src/index.js

```diff
@@ -1,5 +1,13 @@
 const trialDivision = (number) => {
-    
-  }
+  let i = 1;
+  let j = 0;
+  while(i <= number ) {
```

Comment:
Sabes aquí el número de ciclos exactos que vas a realizar, por lo que te recomiendo usar un ciclo for.